salt/cloud/clouds/scaleway.py

```diff
@@ -313,16 +314,22 @@ def __query_node_data(server_name):
     return ret
 
 
-def query(method='servers', server_id=None, command=None, args=None,
-          http_method='get'):
+def query(method='servers', root='api_root', server_id=None, command=None, args=None,
```

Comment:
We do not break stuff where people use positional args, so this `root` needs to be moved to the end of the function definition.

Reply:
sure, thanks for the feedback